### DISCUSSION TITLE
Impossible to run aqm2 server

### DISCUSSION THREAD

**Discussion by Stonurd** [itzg/docker-minecraft-server#2498] [Q&A]
Hi, I learned how to make a container to run Minecraft server, I have an absurd problem. This whole day I tried to run this fabric modpack (aqm2).
The problem is basically 1/10 mods cause "java.util.zip.ZipException: zip END header not found" individually, so the log is beyond 7000 lines.

My last try docker compose file code is:

> version: "3.8"
> 
> services:
>   minecraft:
>     # Container Image on dockerhub 
>     image: "itzg/minecraft-server"
>     tty: true
>     stdin_open: true
>     # Port Mappings
>     ports:
>       - "25565:25565"
>     # Environment Variables
>     environment:
>       # call for curseforge servertype
>       TYPE: "AUTO_CURSEFORGE"
>       #api key from https://console.curseforge.com/#/login
>       CF_API_KEY: "$$2a$$10$$KJ8VQUSoEe1dOAAUCSIduel63.KYwgHwpLA1KQDGI8TWf2plenrg2"
>       #link of the modpack look for specific version or auto update
>       CF_PAGE_URL: "https://www.curseforge.com/minecraft/modpacks/another-quality-modpack-2/files/4709736"
>       #CF_EXCLUDE_MODS: "437443"
>       MEMORY: "4G"
>       MAX_MEMORY: "8G"
>       VERSION: "1.19.2"
>       DIFFICULTY: "hard"
>       CF_PARALLEL_DOWNLOADS: "1"
>       EULA: "TRUE"
>     volumes:
>       # attach the relative directory 'data' to the container's /data path
>       - R:\MCSERVER:/data
>     # restart policy - attempt to restart 1 times if the container fails
>     restart: "on-failure:no"

**The log is attached, please, Make some expert people save me.**
[logs.txt](https://github.com/itzg/docker-minecraft-server/files/13401176/logs.txt)

**Comment by itzg** [comment]
I'm wondering if some of the files got corrupted during a download. You could manually clean those out and let it redownload by running:

```
docker compose run --rm --entrypoint bash minecraft -c 'rm mods/*.jar'
```
